### PR TITLE
[expo] Add barcode types constants

### PR DESCRIPTION
**Pull request by fhelwanger**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: https://docs.expo.io/versions/latest/sdk/bar-code-scanner.html#barcodetypes-arraystring--an-array-of-bar-code-types-usage-barcodescannerconstantsbarcodetypecodetype-where-codetype-is-one-of-the-listed-above-default-all-supported-bar-code-types-for-example-barcodetypesbarcodescannerconstantsbarcodetypeqr
- [X] Increase the version number in the header if appropriate.
- [X] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/expo/index.d.ts [hunk 1]
```diff
@@ -632,7 +633,32 @@ export interface BarCodeScannerProps extends ViewProperties {
     onBarCodeRead?: BarCodeReadCallback;
 }
 
-export class BarCodeScanner extends Component<BarCodeScannerProps> { }
+export class BarCodeScanner extends Component<BarCodeScannerProps> {
+  static Constants: {
```
Comment (KonstantinKai) [inline types/expo/index.d.ts]: @fhelwanger hello, https://github.com/expo/expo-sdk/blob/master/src/CameraBasedBarCodeScanner.js#L19
BarCodeScanner Constants is extending CameraConstants. Can you extend or union this interface with CameraConstants? Like this -> https://github.com/DefinitelyTyped/DefinitelyTyped/pull/23812/commits/cec29159bd0034cbdc527d8e60b2eef1f153ade1#diff-61992a697c86241af33a41d9361c703c

**Review by KonstantinKai — CHANGES_REQUESTED** [review]

File: types/expo/index.d.ts (hunk 1 above)
Comment (KonstantinKai) [reply types/expo/index.d.ts]: and interface props should separate by semicolon

Comment (fhelwanger) [reply types/expo/index.d.ts]: Hello! Sure, done.

**Review by KonstantinKai — APPROVED** [review]
@fhelwanger approved 👍

**Comment by fhelwanger** [comment]
Already done in #23812

**Pull request closed without merging**